### ISSUE TITLE
Zict kivy error

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [ContinuumIO/anaconda-issues#7956]
ran command 

> conda install -c krisvanneste kivy 

got error

> UnsatisfiableError: The following specifications were found to be in conflict:

> zict
> kivy
> Use "conda info " to see the dependencies for each package.

**Comment by thegeekorthodox** [comment]
I just ran into this problem. Were you able to get it worked out?

**Comment by Abdur-rahmaanJ** [comment]
no

**Comment by Nebelhom** [comment]
I encountered the same issue. Has anyone found a solution to this problem?

Thanks :)

**Comment by thegeekorthodox** [comment]
yeah I eventually got fed up with it, removed everything, rebooted, and reinstalled everything. I ran the exact same commands, and it worked.

**Comment by Abdur-rahmaanJ** [comment]
weird
